src/ui/component/wunderbar/view.jsx

```diff
@@ -96,9 +96,9 @@ class WunderBar extends React.PureComponent<Props, State> {
 
     const query = value.trim();
     const showSnackError = () => {
-      doShowSnackBar({
-        message: __('Invalid LBRY URL entered. Only A-Z, a-z, 0-9, and "-" allowed.'),
-      });
+      doShowSnackBar(
+        <span>{'Invalid LBRY URL entered. Only A-Z, a-z, 0-9, and "-" allowed.'}</span>
```

Comment:
This doesn't need to have a `<span>` wrapped around it. 

Also now that I'm thinking about it. We can use `isError: true`, to get the error snack bar.